controllers/comments.js

```diff
@@ -127,29 +120,105 @@ export default {
             createdAt: comment.createdAt,
             updatedAt: comment.updatedAt,
             body: comment.body,
-            highlitedText: comment.highlitedText,
+            highlightedText: comment.hightedText,
             author,
             likes: getLikes.count
           };
         })
       );
-      // console.log(structuredComments);
+      // console.log(formatedComments);
       return res.status(200).json({
         status: res.statusCode,
-        comments: structuredComments,
-        commentsCount: structuredComments.length
+        comments: formatedComments,
+        commentsCount: formatedComments.length
       });
     } catch (e) {
-      if (e.message) {
-        return res.status(500).json({
+      return res.status(500).json({
+        status: res.statusCode,
+        error: e.message || '',
+        message: 'Something happened on the server'
+      });
+    }
+  },
+
+  /**
+   * An author of a comment is allowed to update a comment
+   * When a comment is updated we keep the  old comment edits
+   * so that a user can be able to see edit history
+   * @param {object} req
+   * @param {object} res
+   * @return {object} res
+   *
+   */
+  update: async (req, res) => {
+    const { body } = req.body;
+    const { slug, commentId } = req.params;
+    const { id } = req.user;
+
+    if (!body || body.trim() === '') {
```

Comment:
I think it is valid in some scenarios where someone can decide to leave a comment of a number. If it is in that case, this body.trim() is going to throw an error. You can think about that.

Reply:
The body will be sent as text, not an integer.